### PR TITLE
Implement Unit Test #8

### LINKED ISSUE #8: Add some unit tests
Please create unit tests for all classes which **implement** an interface. You need only to test **methods which return a value**. Don't create tests for any other methods.

Please refer to the following pull request as for an example: https://github.com/extsoft/jwtabc-2018/pull/2/files

### PR DESCRIPTION
**Pull request by akdcstar**
Closes #8

### CODE REVIEW

**Comment by extsoft** [comment]
@akdcstar please merge or rebase latest master into this branch. And solve a conflict with the Selenium dependency.

**Comment by akdcstar** [comment]
@extsoft I've merged current branch with origin/master, please check.

**Comment by extsoft** [comment]
> @extsoft I've merged current branch with origin/master, please check.

@akdcstar great!

File: src/test/java/net/comments/sample/AttentiveStudentTest.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import org.testng.annotations.Test;
+
+public class AttentiveStudentTest {
+
+    @Test
+    public void checkName() {
+        MatcherAssert.assertThat(new AttentiveStudent("Oleg").getName(), Matchers.is("Oleg"));
+    }
+
+}
```
Comment (extsoft) [inline src/test/java/net/comments/sample/AttentiveStudentTest.java]: Please add an empty line at the end of the file.

File: src/test/java/net/comments/sample/BadStudentTest.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+public class BadStudentTest {
+
+    @Test
+    public void checkName() {
+        MatcherAssert.assertThat(new BadStudent("Ivan").getName(), Matchers.is("Ivan"));
+    }
+
+
+}
```
Comment (extsoft) [inline src/test/java/net/comments/sample/BadStudentTest.java]: Please add an empty line at the end of the file.

File: src/test/java/net/comments/sample/LimitedRoomTest.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LimitedRoomTest {
+
+    @Test
+    public void testGetPersonInRoom() {
+        List<Person> personList = new ArrayList<Person>();
```
Comment (extsoft) [inline src/test/java/net/comments/sample/LimitedRoomTest.java]: Why didn't you use `LimitedRoom` class here?

File: src/test/java/net/comments/sample/SubjectCookingTest.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+import org.hamcrest.Matchers;
+import org.testng.annotations.Test;
+
+public class SubjectCookingTest {
+
+    @Test
+    public void checkSubjectName() {
+        MatcherAssert.assertThat(new SubjectCooking().getSubjectName(), Matchers.is("Bake a Cake"));
+    }
+}
```
Comment (extsoft) [inline src/test/java/net/comments/sample/SubjectCookingTest.java]: Please add an empty line at the end of the file.

File: src/test/java/net/comments/sample/SubjectGitTest.java [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+import org.hamcrest.Matchers;
+import org.testng.annotations.Test;
+
+public class SubjectGitTest {
+
+    @Test
+    public void checkSubjectName() {
+        MatcherAssert.assertThat(new SubjectGit().getSubjectName(), Matchers.is("Use Git"));
+    }
+}
```
Comment (extsoft) [inline src/test/java/net/comments/sample/SubjectGitTest.java]: Please add an empty line at the end of the file.

File: src/test/java/net/comments/sample/TechTeacherTest.java [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+import org.hamcrest.Matchers;
+import org.testng.annotations.Test;
+
+public class TechTeacherTest {
+
+    @Test
+    public void checkName() {
+        MatcherAssert.assertThat(new TechTeacher("Egor").getName(), Matchers.is("Egor"));
+    }
+}
```
Comment (extsoft) [inline src/test/java/net/comments/sample/TechTeacherTest.java]: Please add an empty line at the end of the file.

File: src/test/java/net/comments/sample/UnlimitedRoomTest.java [hunk 7]
```diff
@@ -0,0 +1,23 @@
+package net.comments.sample;
+
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.collection.IsIterableContainingInOrder;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class UnlimitedRoomTest {
```
Comment (extsoft) [inline src/test/java/net/comments/sample/UnlimitedRoomTest.java]: Why `UnlimitedRoom` is not used in the test?

**Review by extsoft — CHANGES_REQUESTED** [review]
@akdcstar Please pay attention to the comments below.

**Comment by akdcstar** [comment]
@extsoft I've fixed all your comments

**Review by extsoft — APPROVED** [review]
@akdcstar look's great!

**Pull request merged**